### DISCUSSION TITLE
Routed Connections - What does it mean ?

### DISCUSSION THREAD

**Discussion by netcedec** [strongswan/strongswan#2134] [Q&A]
What this means when outputting the "ipsec statusall"

![image](https://github.com/strongswan/strongswan/assets/150895654/2cfb0395-6817-4aea-8417-5f328fdba356)

**Comment by tobiasbrunner** [comment]
The first section lists configured connections, the second trap policies installed based on (some of) these configs in the kernel (called routed for historical reasons). Trap policies are installed either automatically via `auto=route` or manually via `ipsec route`. Note that all this is different when using `swanctl` or vici instead of the legacy ipsec.conf-based backend.

**Reply by netcedec** [reply]
Does this section only have entries when Phase1 is installed?

**Reply by tobiasbrunner** [reply]
No, both sections are based on the config and only locally relevant. The actual connections (established or connecting) are listed in a separate section called `Security Associations`.
